packages/checkPrescriptionStatusUpdates/src/checkPrescriptionStatusUpdates.ts

```diff
@@ -58,31 +58,42 @@ const lambdaHandler = async (event: APIGatewayProxyEvent): Promise<APIGatewayPro
     exclusiveStartKeyPrescriptionID: event.headers["exclusivestartkey-prescriptionid"],
     exclusiveStartKeyTaskID: event.headers["exclusivestartkey-taskid"]
   }
-  const queryResult = await getItemStatusUpdates(inputData, logger)
 
-  let statusCode = 200
   const result = {
     items: []
   }
-  if (queryResult.Count === 0) {
-    statusCode = 404
-  } else {
-    result.items = queryResult.Items
-  }
-
   const headers = {
     "Content-Type": "application/json",
     "Cache-Control": "no-cache"
   }
 
-  if (queryResult.LastEvaluatedKey) {
-    for (const key in queryResult.LastEvaluatedKey) {
-      headers[`LastEvaluatedKey-${key}`] = queryResult.LastEvaluatedKey[key]
+  const MIN_RESULTS_RETURNED = 5
```

Comment:
These could be made into lambda env vars in case any fussy suppliers want them adjusting?

Reply:
8490dff8f31ee055382be9b9615316ec87f22c89